addons/mail/models/mail_activity.py

```diff
@@ -389,6 +392,35 @@ def _compute_display_name(self):
             name = record.summary or record.activity_type_id.display_name
             record.display_name = name
 
+    # ------------------------------------------------------
+    # Notifications
+    # ------------------------------------------------------
+
+    def _notify_owner_store(self, store, notification_type="mail.record/update_activity"):
+        self.ensure_one()
```

Comment:
Why ensure one? We call this method in a loop. Allowing batches will improve performances (since we can batch fetching of `owner_model`.